recipe/conda_build_config.yaml

```
@@ -1,2 +1,8 @@
-MACOSX_SDK_VERSION:        # [osx and x86_64]
-  - 10.12                  # [osx and x86_64]
+MACOSX_SDK_VERSION:  # [osx and x86_64]
+  - 10.12            # [osx and x86_64]
```

```suggestion
  - 10.13            # [osx and x86_64]
```
Should this be consistent?